src/functions/helpers.ts

```diff
@@ -126,6 +161,59 @@ export function toJsDate(value: string | number | boolean | null | undefined): D
   return numberToJsDate(toNumber(value));
 }
 
+export enum ABSOLUTE_RELATIVE_MODE {
+  ROW_ABSOLUTE_COL_ABSOLUTE = 1,
+  ROW_ABSOLUTE_COL_RELATIVE = 2,
+  ROW_RELATIVE_COL_ABSOLUTE = 3,
+  ROW_RELATIVE_COL_RELATIVE = 4,
+}
+
+export function toA1Notation(
+  rowNumber: number,
+  colNumber: number,
+  absoluteRelativeMode: number = ABSOLUTE_RELATIVE_MODE.ROW_ABSOLUTE_COL_ABSOLUTE
```

Comment:
Is there any reason not to use the `toXC` function ?
Then, if you really need it, you should maybe keep it "simple" with an object like `{rowFixed: boolean, colFixed: boolean}` as it's done in other part of the code (e.g. in range)

Reply:
@anhe-odoo Thanks! No special reason - simply because I don't remember we have this helper 😅 I'll use `toXC()` instead.
I'm not sure whether we should add `A1Notation` in all related existing method names. For now we only use R1C1 in `ADDRESS` so IMHO it's not necessary.